client/assets/script/main.js

```diff
@@ -0,0 +1,33 @@
+    // login / register / logout
```

Comment:
semua event listener sebaiknya didaftarkan ketika dom nya ready
menurut http://tutorials.jenkov.com/jquery/document-ready.html :
_"Whenever you use jQuery to manipulate your web page, you wait until the document ready event has fired. The document ready event signals that the DOM of the page is now ready, so you can manipulate it without worrying that parts of the DOM has not yet been created. The document ready event fires before all images etc. are loaded, but after the whole DOM itself is ready."_